lldb/source/Interpreter/OptionGroupValueObjectDisplay.cpp

```diff
@@ -58,7 +58,12 @@ static const OptionDefinition g_option_table[] = {
     {LLDB_OPT_SET_1, false, "element-count", 'Z',
      OptionParser::eRequiredArgument, nullptr, {}, 0, eArgTypeCount,
      "Treat the result of the expression as if its type is an array of this "
-     "many values."}};
+     "many values."},
+    {LLDB_OPT_SET_1, false, "bind-generic-types", /* no short option */ 1,
+     OptionParser::eRequiredArgument, nullptr, {}, 0,
+     eArgTypeBoolean, "Controls whether to bind the generic type parameters " 
+       " before evaluating an expression. Defaults to true."}
```

Comment:
How about: `Controls whether any generic types in the current context should be bound to their dynamic concrete types before evaluating.`